Tests/YCoreUITests/Assets/Strings/en.lproj/Settings.strings

```diff
@@ -0,0 +1,9 @@
+/* 
+   Settings.strings
+  
+  Created by Panchami Shenoy on 27/10/22.
+  
+*/
+"string_value" = "hello";
+"int_value" = "123";
+"bool_value" = "true false";
```

Comment:
Let's use Title_Case examples that seem more like how we'd recommend people to use localized string keys.
```suggestion
"Settings_Title" = "Settings";
"Settings_Font" = "Font";
"Settings_Color" = "Color";
```